libs/wingsdk/src/fs/fs.ts

```diff
@@ -461,6 +610,37 @@ export class Util {
     fs.symlinkSync(target, path, type);
   }
 
+  /**
+   * Match files using the patterns the shell uses.
+   *
+   * Built with the great `glob` package, based on https://www.npmjs.com/package/glob
+
+   * @param pattern The pattern to match.
+   * @param options Glob options.
+   * @returns List of matching files.
+   */
+  public static glob(pattern: string, options: GlobOptions = {}): string[] {
+    return glob.sync(pattern, options);
+  }
+
+  /**
+   * Calculate an MD5 content hash of all the files that match a glob pattern.
+   *
+   * @param dir The root directory.
+   * @param globPattern The glob pattern to match (defaults to all files and subdirectories).
+   * @returns An md5 hash of the file contents.
+   */
+  public static md5(dir: string, globPattern: string = "**/*") {
+    const hash = crypto.createHash("md5");
+    const files = this.glob(globPattern, { nodir: true, cwd: dir });
+    for (const f of files) {
+      const data = fs.readFileSync(this.join(dir, f));
+      hash.update(data);
+    }
+
+    return hash.digest("hex");
+  }
```

Comment:
md5 taking a directory as a first argument seems like a secondary use case / kind of API, no? Normally I'd expect to directly pass the information I expect. Maybe we can leave this out of the PR...

Reply:
This is `fs.md5()`, not `util.md5()`. Would that make sense more sense?

Reply:
ah, got it. Fair enough